F-FrontEnd/gnu_module/export_module.c

```diff
@@ -429,23 +429,21 @@ struct list_node *cons_list(x,l)
     return(lp);
 }
 
-expr list2(code,x1,x2)
-     enum expr_code code;
-     expr x1,x2;
+expr list2(enum expr_code code,
+	     expr x1,
+		 expr x2)
+
 {
     return(make_enode(code,(void *)cons_list(x1,cons_list(x2,NULL))));
 }
 
-expr list3(code,x1,x2,x3)
-     enum expr_code code;
-     expr x1,x2,x3;
+expr list3(enum expr_code code, expr x1,expr x2,expr x3)

```

Comment:
NIT: Missing some whitespaces. 
```suggestion
expr list3(enum expr_code code, expr x1, expr x2, expr x3)
```

Reply:
Reverted changes to this file